.chloggen/codeboten_doc-status.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: mdatagen
```

Comment:
I miss readme.md changes showing how this can be used by component developers.

Reply:
> I miss readme.md changes showing how this can be used by component developers.

Just confirming here @jpkrohling do you mean changelog.md changes?